### ISSUE TITLE
Photo API?

### ISSUE THREAD

**Issue by dyoung418** [mackliet/church_of_jesus_christ_api#9]
Does the church have an endpoint for downloading a photo of a member that could be added to the API?

**Comment by mackliet** [comment]
I'll try to take a look at it sometime this week, there must be something.

**Comment by mackliet** [comment]
I added a download_member_photo method, it returns None if no photo is available, or a bytes object for the photo as a JPEG

**Comment by dyoung418** [comment]
Thank you!

**Comment by dyoung418** [comment]
I might be doing something wrong, but I'm unable to find this API endpoint.  The documentation site doesn't list it and if I look at the code, I think it is supposed to be api.download_member_photo(), but in trying that, i get an error that it doesn't exist.

**Comment by mackliet** [comment]
Good call, I forgot to update the docs. I should add a git hook or CI/CD check or something for that...

I tried on my machine 
`pip install church_of_jesus_christ_api --upgrade`

and then in the python console I did this:
```
from church_of_jesus_christ_api import ChurchOfJesusChristAPI as API
api = API(<username>, <password>)
with open("my_photo.jpeg", "wb") as file:
  file.write(api.download_member_photo())
```

And you should be able to see your own photo in your current working directory

**Comment by dyoung418** [comment]
Works great.  Thank you!

**Comment by connwork** [comment]
Hi @mackliet 

Is there a way to use this for multiple people? Using something like this isn't working for me: 

`file.write(api.download_member_photo(member_id=xxxxxxxxxxx))`
